service/mapper/transitions.go

```diff
@@ -92,16 +94,34 @@ func (t *Transitions) BootstrapState(s *State) error {
 		return fmt.Errorf("invalid status for bootstrapping state (%s)", s.status)
 	}
 
-	skipBoostrap := true
-	if skipBoostrap {
-		log.Info().Msgf("skip bootstrap")
-		s.status = StatusResume // StatusResume will save the height
-		return nil
+	first, err := t.chain.Root()
```

Comment:
This fixes an issue that the default height was set to MaxUint, which caused the registers from the checkpoint to be indexed with a wrong height.

The correct root height needs to be initialized here